server/game/cards/14.1-RaW/AkodoReserveCompany.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes, Locations } = require('../../Constants');
+const AbiltiyDsl = require('../../abilitydsl');
+
+class AkodoReserveCompany extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Bow an attacking character',
+            condition: () => this.game.isDuringConflict() && this.game.allCards.some(card => card.hasTrait('battlefield') && card.location.includes(Locations.PlayArea)),
+            target: {
+                cardType: CardTypes.Character,
+                cardCondition: (card, context) => card.isParticipating() && card.controller === context.player,
+                gameAction: AbiltiyDsl.actions.multiple([
+                    AbiltiyDsl.actions.moveToConflict(context => ({ target: context.source })),
+                    AbiltiyDsl.actions.sendHome()
```

Comment:
You'll probably have to implement this as a "then" block.
![image](https://user-images.githubusercontent.com/60416666/76157959-fbfa3200-60d5-11ea-8fb7-7eedbbe72ef1.png)